package.json

```diff
@@ -21,23 +21,24 @@
   },
   "license": "MIT",
   "dependencies": {
-    "async": "~0.9.0",
-    "commondir": "0.0.1",
-    "debug": "^2.0.0",
-    "lodash": "^3.6.0",
-    "semver": "^4.1.0",
-    "toposort": "^0.2.10"
+    "async": "^0.9.2",
+    "bluebird": "^3.3.5",
+    "commondir": "1.0.1",
```

Comment:
Missing `^` operator, is that intentional? I mean now you require exactly version `1.0.1` and nothing else.